src/platform/forms-system/src/js/components/ContactInfoReview.jsx

```diff
@@ -48,28 +48,34 @@ const ContactInfoReview = ({ data, editPage, content, keys }) => {
   const isUS = address.addressType !== ADDRESS_TYPES.international;
   const stateOrProvince = isUS ? 'state' : 'province';
 
+  const errorMsg = errorMessage => (
+    <span className="usa-input-error-message">{errorMessage}</span>
+  );
+
   // Label: formatted value in (design) display order
   const display = [
     [content.homePhone, () => renderTelephone(homePhone)],
     [content.mobilePhone, () => renderTelephone(mobilePhone)],
+    [content.email, () => email || errorMsg(content.missingEmailError)],
+    [content.country, () => address.countryName],
```

Comment:
Does this mean that `country` is an optional field?

Reply:
Country is set to `'United States'` by default, so it's always defined; but again, given the issues we had with zip code, probably better to show an error.